app/components/public/ticket-list.js

```diff
@@ -1,20 +1,27 @@
 import Ember from 'ember';
 import FormMixin from 'open-event-frontend/mixins/form';
-import { sumBy } from 'lodash';
 
 const { Component, computed } = Ember;
 
 export default Component.extend(FormMixin, {
   classNames: ['ui', 'segments', 'ticket-list'],
 
   hasTicketsInOrder: computed('tickets.@each.orderQuantity', function() {
-    return sumBy(this.get('tickets'), 'orderQuantity') > 0;
```

Comment:
@niranjan94 well.. it doesn't seem to be  working throws a  *cannot  use get for undefined* error